src/test/java/com/puppycrawl/tools/checkstyle/TreeWalkerTest.java

```diff
@@ -190,20 +190,15 @@ public void testImproperFileExtension() throws Exception {
             final String content = "public class Main { public static final int k = 5 + 4; }";
             writer.write(content);
         }
-        final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
-        verify(checkConfig, file.getPath(), expected);
+        execute(checkConfig, file.getPath());
```

Comment:
Pmd is right, there is no validation of execution.
Please make regular input file with header comment,  and in test copy it to temp folder and rename or create temp file and copy to it input file from repo.

Reply:
@romani done